### PR TITLE
doc: improve upgrading documentation

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Comment by jd** [comment]
Pull-request updated, HEAD is now d35289f13910ffca5a0d9a0b02ccce3d20f9499a

File: doc/source/install.rst [hunk 1]
```diff
@@ -152,22 +152,32 @@ and storage:
 Upgrading
 =========
 In order to upgrade from a previous version of Gnocchi, you need to make sure
-that your indexer and storage are properly upgraded. Run the following:
+that your indexer and storage are properly upgraded.
+
+.. warning::
+
+   Upgrade is only supported between one major version to another or between
+   minor version: e.g version 2.1 to version 2.2 or version 2.1 to version 3.0
```
Comment (chungg) [inline doc/source/install.rst]: slightly confusing but does this mean we don't support 2.0->2.2?

**Comment by jd** [comment]
Pull-request updated, HEAD is now 3935e33c7b768ee9e3f83e16c0220c6e6e95e323

**Comment by jd** [comment]
Pull-request updated, HEAD is now 96834e3f8cacffe8cf1f7c40f7e3dac7a5472703

**Comment by jd** [comment]
Pull-request updated, HEAD is now 55f4e71793fc153878ee28e5b19e19c763f9aeb1

**Comment by jd** [comment]
Pull-request updated, HEAD is now c2012ac3d603154d8b81d0a9f1abb50a1cc06a7a

**Comment by jd** [comment]
Pull-request updated, HEAD is now 99cfcd58830d80da20926f1e4739f04e245526eb

**Comment by jd** [comment]
Pull-request updated, HEAD is now e037b9be3489cf172288713356be8a40011af334

**Pull request merged**